application/console/controllers/GaController.php

```diff
@@ -43,7 +43,8 @@ public function actionRegister_event()
 
         $gaId = \Yii::$app->params['googleAnalytics']['measurementId'];
 
-        print_r(PHP_EOL .
+        print_r(
```

Comment:
print_r is ugly, but since it was already there... Otherwise, I don't see why an echo or print isn't sufficient in this case.

Reply:
Good to know.